---
layout: archive
title: ""
permalink: /teaching/
author_profile: true
---


<!DOCTYPE html>
<html lang="en">
<head>
  <style>
    .profile-container {display: flex; 
                        align-items: flex-start;
                        gap: 30px;
                        margin-bottom: 20px;}
    .photo-container {flex: 0 0 180px;}
    .photo-container img {width: 180px;
                          height: 240px;
                          object-fit: cover;}
    .info-container {flex: 1;}
    .label {font-weight: bold;}
    .bio {margin-top: 20px;}
  </style>
</head>
<body>
<h2>MERCI</h2>
<p>Mitochondrial-enabled reconstruction of cellular interactions</p>
<hr>
<div class="profile-container">
  <!-- 左侧照片区域 -->
  <div class="photo-container">
    <img src="../images/ZhangLab-Photo/MERCI.png" alt="">
  </div>
  <!-- 右侧信息区域 -->
  <div class="info-container">
    <div class="bio">
      <p><span class="label">简介:</span> MERCI (Mitochondrial-enabled reconstruction of cellular interactions) is a statistical deconvolution method for tracing and quantifying mitochondrial trafficking between cancer and T cells. (Zhang, H., Yu, X,. & Li, B. Cancer Cell ,2023).Through rigorous benchmarking and validation, MERCI accurately predicts the recipient cells and their relative mitochondrial compositions. Application of MERCI to human cancer samples identifies a reproducible MT transfer phenotype, with its signature genes involved in cytoskeleton remodeling, energy production, and TNF-α signaling pathways. Moreover, MT transfer is associated with increased cell cycle activity and poor clinical outcome across different cancer types.  MERCI is accessible from https://github.com/shyhihihi/MERCI.</p>
    </div>
  </div>
</div>
</body>
</html>


<!DOCTYPE html>
<html lang="en">
<head>
  <style>
    .profile-container {display: flex; 
                        align-items: flex-start;
                        gap: 30px;
                        margin-bottom: 20px;}
    .photo-container {flex: 0 0 180px;}
    .photo-container img {width: 180px;
                          height: 240px;
                          object-fit: cover;}
    .info-container {flex: 1;}
    .label {font-weight: bold;}
    .bio {margin-top: 20px;}
  </style>
</head>
<body>
<h2>MitoTracer</h2>
<hr>
<div class="profile-container">
  <!-- 左侧照片区域 -->
  <div class="photo-container">
    <img src="../images/ZhangLab-Photo/MitoTracer.png" alt="">
  </div>
  <!-- 右侧信息区域 -->
  <div class="info-container">
    <div class="bio">
      <p><span class="label">简介:</span> Uncovering heterogeneous cell populations in single-cell sequencing datasets has provided valuable insights into the tumor microenvironment and developmental processes. Traditional lineage tracing methods have relied on gene expression and nuclear genome mutations. Recently, researchers have recognized the mitochondrial genome as an ideal natural cell barcode due to its small size and high copy number. Several lineage reconstruction strategies leveraging mitochondrial mutations have been developed for single-cell RNA and/or DNA sequencing data. However, these methods still face limitations, including lower accuracy, lack of an automated pipeline, and incompatibility with all single-cell sequencing platforms. To overcome these challenges, we developed MitoTracer, a fully automated end-to-end computational method for identifying informative mitochondrial mutations across single-cell RNA and DNA sequencing data. This pipeline performs all essential analysis steps, including read mapping, generating a mitochondrial variant allele frequency matrix, selecting informative mitochondrial mutations, and inferring clonal structures with higher accuracy compared to existing methods.</p>
    </div>
  </div>
</div>
</body>
</html>


      
